web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -417,6 +417,9 @@ $.extend(fixmystreet.set_up, {
             } else {
                 $category_meta.empty();
             }
+            if (fixmystreet.assets) {
+                fixmystreet.assets.update_usrn_field();
```

Comment:
I'm not sure what this is for/ how the USRN could change at this point?

Reply:
This is for the situation where you've changed the category and so the report extra fields have been altered in the DOM.